improver/cli/apply_emos_coefficients.py

```diff
@@ -214,25 +214,25 @@ def process(cube: cli.inputcube,
     if input_forecast_type == "probabilities":
         result = ConvertLocationAndScaleParametersToProbabilities(
             distribution=distribution,
-            shape_parameters=shape_parameters).process(
-            location_parameter, scale_parameter, original_current_forecast)
+            shape_parameters=shape_parameters)(
+                location_parameter, scale_parameter, original_current_forecast)
     elif input_forecast_type == "percentiles":
         perc_coord = find_percentile_coordinate(original_current_forecast)
         result = ConvertLocationAndScaleParametersToPercentiles(
             distribution=distribution,
-            shape_parameters=shape_parameters).process(
-            location_parameter, scale_parameter, original_current_forecast,
-            percentiles=perc_coord.points)
+            shape_parameters=shape_parameters)(
+                location_parameter, scale_parameter, original_current_forecast,
+                percentiles=perc_coord.points)
     elif input_forecast_type == "realizations":
         # Ensemble Copula Coupling to generate realizations
         # from the location and scale parameter.
         no_of_percentiles = len(current_forecast.coord('realization').points)
         percentiles = ConvertLocationAndScaleParametersToPercentiles(
             distribution=distribution,
-            shape_parameters=shape_parameters).process(
-            location_parameter, scale_parameter, original_current_forecast,
-            no_of_percentiles=no_of_percentiles)
-        result = EnsembleReordering().process(
+            shape_parameters=shape_parameters)(
+                location_parameter, scale_parameter, original_current_forecast,
+                no_of_percentiles=no_of_percentiles)
+        result = EnsembleReordering()(
```

Comment:
I agree with Fiona that some of the calls with very long lists of "process" arguments would be clearer separated.  So I'll note that on that comment.  But in this particular file (apply EMOS CLI) may I ask you just to leave it as you have, as I'm currently refactoring this anyway so will do some cleanup then.

Reply:
Leaving alone.